src/components/PersonalBlogPosts/index.css

```diff
@@ -0,0 +1,16 @@
+.personalBlogPostsBtn,
+.personalBlogPostsBtn:hover,
+.personalBlogPostsBtn:focus {
+  background: transparent;
+  border: none;
+  color: var(--blog-background-profile);
+  border: 1px solid var(--blog-background-profile);
+}
```

Comment:
it's better to leave the button visible rather than transparent.
we can make it to get darker as another option.